etl/steps/data/garden/oecd/2024-07-01/road_accidents.py

```diff
@@ -0,0 +1,149 @@
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("road_accidents", channel="meadow", version="2024-07-01")
+    ds_passenger = paths.load_dataset("passenger_travel", channel="meadow", version="2024-07-01")
+    ds_old = paths.load_dataset("road_accidents", version="2023-08-11")
+    ds_population = paths.load_dataset("population")
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["road_accidents"].reset_index()
+    tb_old = ds_old["road_accidents"].reset_index()
+    tb_passenger = ds_passenger["passenger_travel"].reset_index()
+
+    # standardize both tables
+    tb_passenger = tb_passenger.rename(columns={"vehicle_type": "measure"})
+
+    # concatenate the two tables
+    tb = pr.concat([tb, tb_passenger], ignore_index=True)
+
+    col_origins = tb["obs_value"].origins.copy()
+
+    # pivot table
+    tb = tb.pivot_table(index=["country", "year"], columns=["measure"], values="obs_value").reset_index()
+
+    # rename passenger travel columns
+    tb = tb.rename(
+        columns={
+            "Rail": "passenger_kms_rail",
+            "Road": "passenger_kms_road",
+            "Passenger cars": "passenger_kms_car",
+            "Buses": "passenger_kms_bus",
+        }
+    )
+
+    # harmonize country names
+    tb = geo.harmonize_countries(df=tb, countries_file=paths.country_mapping_path, warn_on_unused_countries=False)
+    tb_old = geo.harmonize_countries(
+        df=tb_old, countries_file=paths.country_mapping_path, warn_on_unused_countries=False
+    )
+
+    # Combine new and old data
+    tb = tb.merge(tb_old, how="outer", on=["country", "year"], suffixes=("", "_old")).copy_metadata(tb)
+
+    # make all column names lowercase
+    tb.columns = tb.columns.str.lower()
+
+    #
+    # process data - combine indicators and add death per million inhabitants/ per thousand passenger kilometers
+    #
+    # if one column is nan use other column, otherwise use new data
+    tb["accident_deaths"] = tb.apply(lambda x: choose_latest_data(x, "deaths", "fatalities"), axis=1)
+    tb["accident_injuries"] = tb.apply(lambda x: choose_latest_data(x, "injuries", "injured"), axis=1)
+    tb["accidents_with_injuries"] = tb.apply(
+        lambda x: choose_latest_data(x, "accidents_involving_casualties", "crashes"), axis=1
+    )
+
+    # drop old columns
+    tb = tb.drop(columns=["fatalities", "injured", "crashes", "deaths", "injuries", "accidents_involving_casualties"])
+
+    # remove passenger travel by road if it is identical to bus or car column
+    tb["passenger_kms_road"] = tb.apply(lambda x: check_road_passenger_travel(x), axis=1)
+
+    # add death per million inhabitants
+    tb = geo.add_population_to_table(tb, ds_population)
+    tb["deaths_per_million_population"] = (tb["accident_deaths"] / tb["population"]) * 1_000_000
+    # drop population as well as old death per million and per vehicle (these numbers are wrong)
```

Comment:
Why are they wrong?

Reply:
They come from the OECD and don't make sense - death per million inhabitants is off by a factor of 10 and death per million vehicles is somehow 10x smaller than per inhabitant, which does not add up with the vehicle to people ratio.